scripts/prover_profile.sh

```diff
@@ -0,0 +1,24 @@
+
+CIRCUIT=2_2
+
+ZETH_TOOL=./zeth_tool/zeth-tool
+PK=~/dev/neodizk/test_data/zeth/zeth_${CIRCUIT}_pk.bin
+VK=~/dev/neodizk/test_data/zeth/zeth_${CIRCUIT}_vk.bin
```

Comment:
Why do we have these paths here? This implies a given structure on the user's machine. I'd rather use the values in https://github.com/clearmatics/zeth/blob/master/setup_env.sh when possible, or simply document these values. e.g.
```
# VK=~/dev/neodizk/test_data/zeth/zeth_${CIRCUIT}_vk.bin

NEODIZK=<your_path_to_neodizk> # To use this script please, close neodizk https://github.com/clearmatics/neodizk and point to its location here.
VK=$NEODIZK/test_data/zeth/zeth_${CIRCUIT}_vk.bin
```

Reply:
Changed as part of the refactor above.  These values are now more configurable, falling back to defaults.